pkg/cmd/cmd.go

```diff
@@ -56,6 +56,7 @@ func Execute() {
 	rootCmd.AddCommand(InfoCmd)
 	rootCmd.AddCommand(PortForwardCmd)
 	rootCmd.AddCommand(EnvCmd)
+	rootCmd.AddCommand(DocsCmd)
```

Comment:
Looks good, I would add the command into workspace mode as well (inside `SetupRootCommand`) - didn't specify that in the issue but that was the plan originally - makes sense @Tpuljak?